pycqed/instrument_drivers/meta_instrument/LutMans/ro_lutman.py

```diff
@@ -190,37 +240,50 @@ def norm_gauss(x, mu, sigma):
             # 1. Generate Pulse envelopes
             ## Simple pulse
             up_len = self.get('M_length_R{}'.format(res))-gauss_length
-            M = self.add_pulse(amplitude=self.get('M_amp_R{}'.format(res)),
-                               length=up_len,
-                               delay=0,
-                               phase=self.get('M_phi_R{}'.format(res)))
+            M = create_pulse(shape=self.pulse_primitive_shape(),
```

Comment:
I like the construct for this in the mw_lutman a lot better but I consider this out of scope for this PR. 
The thing I like better is that there we get the wf_func <- similar to the pulse_primitive _shape and pass the arguments to that function. 

Furthermore, I don't think we are ever using the gauss_shape @wvlothuizen , did you add this functionality?

Reply:
I didn't want to mess with core functionality not specifically related to the UHF driver here.